website/docs/fun/qotd/advanced/interval.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Interval
+---
+
+This command triggers the Main CC to post a new QOTD on an interval.
+
+For more information about the QOTD system, see [the overview page](overview).
+
+## Trigger
+
+**Type:** `Hourly Interval`<br />
+**Interval:** `24`
+**Channel:** (any channel where YAGPDB has `Send Message` permission)
+
+:::danger
+
+You must set a channel for this command or it will not function. It can be any channel where YAGPDB has `Send Message` permission.
+
+:::
```

Comment:
The custom command interface displays a warning when no channel is set for an interval trigger, so I think this callout is superfluous.

Reply:
I personally disagree. I myself am guilty of hyperfixating on setup instructions and ignoring other errors or warnings
until I've completed setting something up, and only then do I go back through instructions again to see if I've missed
anything. I think it is a necessary callout.

Reply:
Fair enough, happy to keep it in then.